sdk/eventgrid/azure-eventgrid/samples/consume_samples/consume_eventgrid_events_from_service_bus_queue.py

```diff
@@ -13,13 +13,17 @@
     1) SB_CONN_STR: The connection string to the Service Bus account
     3) SERVICE_BUS_QUEUE_NAME: The name of the servicebus account
 """
+
+# Note: This sample would not work on pypy since one of the dependencies
+# of azure-eventhub is not pypy compatible.
```

Comment:
is it meant to be this?
```suggestion
# of azure-servicebus is not pypy compatible.
```

Reply:
EG is pypy compatible - EH/SB are not because of uamqp